src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
```

Comment:
it seems health should always be 100))

```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
  }
```